File: app/src/main/java/com/akexorcist/localizationapp/stackedactivity/Stack1Activity.java
package com.akexorcist.localizationapp.stackedactivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.akexorcist.localizationactivity.ui.LocalizationActivity;
import com.akexorcist.localizationapp.R;

/**
 * Created by Akexorcist on 2/3/2017 AD.
 */

public class Stack1Activity extends LocalizationActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_stack_1);

        findViewById(R.id.btn_change_language).setOnClickListener(openChangeLanguageActivity());
    }

    private View.OnClickListener openChangeLanguageActivity() {
        return view -> startActivity(new Intent(this, Stack2Activity.class));
    }
}

File: app/src/main/java/com/akexorcist/localizationapp/stackedactivity/Stack1Activity.kt
package com.akexorcist.localizationapp.stackedactivity

import android.content.Intent
import android.os.Bundle
import com.akexorcist.localizationactivity.ui.LocalizationActivity
import com.akexorcist.localizationapp.databinding.ActivityStack1Binding

class Stack1Activity : LocalizationActivity() {
    private val binding: ActivityStack1Binding by lazy { ActivityStack1Binding.inflate(layoutInflater) }

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)

        binding.btnBack.setOnClickListener { super.onBackPressed() }
        binding.btnChangeLanguage.setOnClickListener {
            startActivity(Intent(this, Stack2Activity::class.java))
        }
    }
}
